gym/lib/gym/generators/package_command_generator_legacy.rb

```diff
@@ -47,7 +47,7 @@ def temporary_output_path
 
       def appfile_path
         path = Dir.glob("#{BuildCommandGenerator.archive_path}/Products/Applications/*.app").first
-        path ||= Dir[BuildCommandGenerator.archive_path + "/**/*.app"].last
+        path ||= Dir[BuildCommandGenerator.archive_path + "/**/*.app"].first
```

Comment:
the `dsym_path` method fetchs the last `.app.dSYM`

shouldn't there be some consistency ?